src/OpenLoco/Title.cpp

```diff
@@ -190,7 +190,8 @@ namespace OpenLoco::Title
             auto& command = *_sequenceIterator++;
             std::visit(overloaded{
                            [](WaitStep step) {
-                               _waitCounter = step.duration / 4;
+                               // This loop slightly deviates from the original, subtract 1 tick to make up for it.
+                               _waitCounter = (step.duration / 4) - 1;
```

Comment:
I think this may be off by a factor of two. The sequence isn't waiting too long enough to be aligned with the music. I'll check.

Reply:
The original sequence is indeed synced to a few beats. The `/ 4` bit is also missing in the original assembly, which leads me to believe it was introduced for debugging at some point.
```
loc_4443D4:
mov     ax, [esi]
add     esi, 2
mov     _titleWaitCounter, ax
jmp     loc_444475
```
However, even without this division, the sequence is no longer synced. Hmm!